src/app/shared/error.component.ts

```diff
@@ -6,6 +6,7 @@ class RavenErrorHandler extends ErrorHandler {
   handleError(err: any): void {
     if (err) {
       Raven.captureException(err.originalError || err);
+      Raven.showReportDialog();
```

Comment:
Was it seriously that simple?

Reply:
According to https://errortracking.prod-preview.openshift.io/openshift_io/fabric8-launcher-frontend/settings/user-feedback/m yes :smile: